bot/constants.py

```diff
@@ -285,6 +285,14 @@ class Wikipedia:
     total_chance = 3
 
 
+class YouTube:
```

Comment:
I think these are probably best kept in the `Emoji` class, so they're with the rest of the emojis.

Reply:
Sure thing!